stdlib/REPL/src/docview.jl

```diff
@@ -299,8 +301,33 @@ function summarize(io::IO, T::DataType, binding::Binding)
     end
 end
 
-function summarize(io::IO, m::Module, binding::Binding)
-    println(io, "No docstring found for module `", m, "`.\n")
+function find_readme(m::Module)::Union{String, Nothing}
+    isnothing(pathof(m)) && return nothing
+    path = dirname(pathof(m))
+    top_path = pkgdir(m)
+    while true
+        for readme in ["README.md", "readme.md", "Readme.md", "ReadMe.md"]
+            readme_path = joinpath(path, readme)
+            if isfile(readme_path)
+                return readme_path
+            end
+        end
```

Comment:
Is there a reason to do this rather than:
```
for file in readdir(path; join=true, sort=true)
    isfile(file) && basename(lowerscase(file)) == "readme.md" || continue
    return file
end
```

I don't think performance matters here, but missing out on a file called `ReAdMe.Md` would be bad.
Or are such file not recognized by github anyway?

Reply:
`README` (without the `.md`) is common aswell. I don't think I have seen the other variants, just `README.md` and `README`.